### DISCUSSION TITLE
Tank size for Engine Hat

### DISCUSSION THREAD

**Discussion by mapeje** [hatlabs/discussions#129] [Q&A]
How do I calibrate the tank size? In the web-GUI, there is a "Total Volume multiplier" and "Constant Offset? When I look in the code, there seems to be a default tank size of 120M3? So if I have a tank size of 12M2, I add "0.1" as multiplier? What does "offset" means?

**Comment by moj0002** [comment]
I have a tank of 56 liters and setting the multiplier to 0.05600 seems to work, I changed m3 to liters in the SignalK instrument panel
